charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: v2.48.0
-version: 25.8.0
+version: 25.9.0
```

I think bump the patch version shall be fine.

```suggestion
version: 25.8.1
```